umiCollapse.wdl

```diff
@@ -290,37 +321,27 @@ task bamSplitDeduplication {
         String outputPrefix
         Int memory = 24
         Int timeout = 6
-        String method = "directional"
-        Int editDistanceThreshold = 1
     }
 
     parameter_meta {
-        umiLength: "length of umi"
         bamFile: "Bam file from bwaMem containing UMIs of varying lengths"
+        umiLength: "Specifies the start of the output files"
         outputPrefix: "Specifies the start of the output files"
         modules: "Required environment modules"
         memory: "Memory allocated for this job"
         timeout: "Time in hours before task timeout"
-        method: "What method to use to identify group of reads with the same (or similar) UMI(s)?"
```

Comment:
Why was method and editDistanceThreshold removed?

Reply:
Ok, since we were using the default values anyway, this makes no difference and fixes the error you were seeing.

Reply:
I tested again, it's not option of method and editDistanceThreshold value cause the failure, it's the option --paired.
If add --paired option to umi-tools dedup command, then will break postDedupBamQC markduplicates step, same error shown above: "SAMException: Value was put into PairInfoMap more than once."

should we use --paired option? The --paired option in umi_tools dedup command is used when the input BAM file contains paired-end sequencing data. When this option is enabled, the deduplication process considers both reads from a paired-end sequencing pair together, rather than treating them as individual reads. This means that if one read from a pair is marked as a duplicate, both reads from that pair will be marked as duplicates and removed from the output.

So looks like we should add this option but it will break the next step.

Reply:
Now the --paired optiona added